core/types/calls.cc

```diff
@@ -2143,7 +2145,7 @@ class Magic_buildRange : public IntrinsicMethod {
         auto secondArgIsNil = other.isNilClass();
         if (firstArgIsNil) {
             if (secondArgIsNil) {
-                rangeElemType = Types::untypedUntracked();
+                rangeElemType = Types::untyped(Symbols::Magic_UntypedSource_buildArray());
```

Comment:
Why are we using buildArray in the `Magic_buildRange` intrinsic?

Reply:
Bad copy-paste, I'll change to `buildRange`